playbooks/common/openshift-cluster/openshift_metrics.yml

```diff
@@ -5,3 +5,12 @@
   hosts: oo_first_master
   roles:
   - openshift_metrics
+
+- name: OpenShift Metrics
+  hosts: oo_masters:!oo_first_master
```

Comment:
`serial: 1` to ensure it only hits one master at a time since it will be restarting the master.